README.rst

```diff
@@ -14,18 +19,21 @@ The way to interact with anonlink system is via Command Line Tool `anonlink`. Yo
  Identifying Information) locally using `anonlink hash`, generate candidate blocks locally to scale up record linkage 
  using `anonlink block`, create a record linkage job in entity service with `anonlink create-project` etc.
 
-### Installation
+Installation
+============
 Currently manual install:
 
 ```python3
-pip install git+https://https://github.com/data61/anonlink-client
+pip install anonlink-client
 ```
 
-### Documentation
+Documentation
```

Comment:
Create and link to github issue to track this *todo*

Reply:
Issue #20 is created